src/Message/CancelSubscriptionRequest.php

```diff
@@ -100,6 +105,27 @@
  */
 class CancelSubscriptionRequest extends AbstractRequest
 {
+    /**
+     * Gets the reason the subscription was canceled
+     *
+     * @return null|string
+     */
+    public function getCancelReason()
+    {
```

Comment:
I've been told a couple times to use type hints instead of docblock comments to specify return values, so that might be nice here (though admittedly inconsistent with the rest of this file)

Reply:
Unless we have a different style guide for this repo at the moment and we just want to be consistent!

Reply:
See comment here https://github.com/vimeo/omnipay-vindicia/pull/61#discussion_r314784176